integration_tests/rest_api_utils.py

```diff
@@ -987,6 +987,34 @@ def ensure_llm_task_response_is_correct(
         assert re.search(response_text_regex, response["output"]["text"])
 
 
+def ensure_llm_task_stream_response_is_correct(
+    response: str,
+    required_output_fields: Optional[List[str]],
+    response_text_regex: Optional[str],
+):
+    print("raw response", response)
```

Comment:
remove this?

Reply:
we can; it's been helpful for debugging if the tests fail